src/types/Users.tsx

```diff
@@ -0,0 +1,28 @@
+interface Geo {
+  lat: string;
+  lng: string;
+}
+
+interface Address {
+  street: string;
+  suite: string;
+  city: string;
+  zipcode: string;
+  geo: Geo;
+}
+
+interface Company {
+  name: string;
+  catchPhrase: string;
+  bs: string;
+}
+
+export interface Users {
```

Comment:
```suggestion
export interface User {
```
Ти в цьому інтерфейсі описуєш один об'єкт user, а не багато. Тому слід використовувати однину